spec/client_spec.js

```diff
@@ -0,0 +1,27 @@
+describe("Client", function() {
+
+    var GiantSwarm = require('../lib/client');
+
+    beforeEach(function() {
+      GiantSwarm.setApiEndpoint('https://api.giantswarm.io');
+      GiantSwarm.setAuthToken(null);
+    });
+
+    it("should allow me to ping the server", function(done){
+      GiantSwarm.ping(function(){
+        var value = 1;
+        expect(value).toEqual(1);
+        done();
+      });
+    });
```

Comment:
It would probably be better to support the error callback here as well, to have a clean error just in case. Otherwise this test will cause yasmine to only produce a timeout error after 5 seconds. This would slow the whole testsuite down when things go terribly wrong.